dcm4che-image/src/main/java/org/dcm4che3/image/PhotometricInterpretation.java

```diff
@@ -111,6 +111,11 @@ else if (tsuid.endsWith("91") || tsuid.endsWith("93"))
         public ColorModel createColorModel(int bits, int dataType, Attributes ds) {
             return ColorModelFactory.createYBRFullColorModel(bits, dataType, ds);
         }
+        
```

Comment:
Would brake decompression of YBR_FULL RLE (US) images

Reply:
You're right. I forgot this specific case.

However, I think YBR_FULL + RLE should be handled as an exception and all other JPEG decompressions must have an RGB as a color model for the output image.

A much more common case, you compress an image into JPEG Baseline with dcmtk and then decompress it with dcm4chee and you will get an image with a wrong color model (YBR_FULL instead RGB).

dcmtk: dcmcjpeg --encode-baseline color-raw.dcm color-jpeg.dcm
dcm4chee: dcm2dcm color-jpeg.dcm color-raw.dcm

![weasis v3 0 3-snapshot_014](https://user-images.githubusercontent.com/993975/45736442-42f71d80-bbeb-11e8-936c-207e00b889b5.png)

Reply:
Will think about to make the Photometric Interpretation of decompressed color images configurable by  ImageWriterFactory.properties.

Reply:
Yes, this will allow having a specific behavior according to the selected decompressor.